README.md

```diff
@@ -543,9 +542,13 @@ This profile is enabled by the `ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS` CMake option.
 ### System Dependencies
 To run a human dynamics estimation scenario, we need a Windows machine to install the Xsens suit SDK for getting the sensory information of the human motions from [Xsens](https://www.xsens.com/) and [ESD USB CAN driver](https://esd.eu/en/products/can-usb2) to get the FTShoes/FTSkShoes sensory information. Refer to [Xsens](#xsens) and [FTShoes/FTSkShoes](#shoes) for more information about the dependencies.
 
-### Configuration
+### System Dependencies
```

Comment:
For some reason the `System Dependencies` has been duplicated.

Reply:
My bad, the information in this replicated System Dependencies previously were in the **Configuration** section (even if they refering to system dependency). I think we can remove that part.

Reply:
removed https://github.com/robotology/robotology-superbuild/pull/257/commits/da2baa151f9f0d933c7023c862228c1dda4ff34c